### PR TITLE
Third PR Updated TEAM.md

### PR DESCRIPTION
**Pull request by farivasf11**
## Attach Screenshot

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

File: docs/TEAM.md [hunk 1]
```diff
@@ -36,10 +36,16 @@
...
 | <img src="./images/team/shekharranjan.jpg" width="80px">                                 | [Shekhar Ranjan](https://github.com/shekhar10feb)            | I'm from India, have basic knowledge of Java, SQL, HTML, CSS and still learning JavaScript, want to learn more and explore advance stuffs.                                                                                                                                                                                                                                                                 |
 | <img src="https://github.com/chiomavera.png?size=80">| [Chioma Vera Nkanmuo](https://github.com/chiomavera)| Hi,I'm Chioma Vera Nkanmuo, Beginner Web developer from Nigeria with basic knowledge of  HTML and  CSS. I am currently in my penultimate level in university. Learning while contributing in making twindle a success.  |
+>>>>>>> b619046436d5ba6b684256ffe061e1f4f24bde4e
```
Comment (RafaelBatman55) [inline docs/TEAM.md:46]: image

Something went wrong, please check again your PR.

**Review by RafaelBatman55 — CHANGES_REQUESTED** [review]

**Comment by johnjacobkenny** [comment]
Conflict symbols are still there

**Review by farivasf11 — COMMENTED** [review]
Ready

File: docs/TEAM.md [hunk 2]
```diff
@@ -3,6 +3,45 @@
...
 | <img src="./images/team/pranav.jpg" width="80px">                          | [Pranav Goel](https://github.com/pranavgoel29)               | I'm from India, Have basic knowledge of HTML, CSS, C++ and C, I'm currently learning Javascript.                                                                                                                                                                                                                                                                                                           |
+|                                                                                          | [Adewale Abiola](https://github.com/princehaybee)            | I am a Nigerian. I have basic knowledge of HTML,CSS and Javascript.                                                                                                                                                                                                                                                                                                                                        |
```
Comment (RafaelBatman55) [inline docs/TEAM.md:6]: You copied this block twice (check from line 45).

You can see many members twice in this table.

File: docs/TEAM.md [hunk 3]
```diff
@@ -3,6 +3,45 @@
...
+| <img src="https://github.com/chiomavera.png?size=80">| [Chioma Vera Nkanmuo](https://github.com/chiomavera)| Hi,I'm Chioma Vera Nkanmuo, Beginner Web developer from Nigeria with basic knowledge of  HTML and  CSS. I am currently in my penultimate level in university. Learning while contributing in making twindle a success.  |
+|                                                                                          | [Alexis Rivas](https://github.com/farivasf11)                 | Hello I'm from Mexico, i just graduated form Computer Systems Engineer and I'm a newbie in the Open Source projects, so I hope contribute with everything I can, and learn by the way             |
```
Comment (RafaelBatman55) [inline docs/TEAM.md:44]: Try to add your profile picture (Check #492)

**Note:** Size must be 80px.

**Review by RafaelBatman55 — CHANGES_REQUESTED** [review]
Please, check again @farivasf11

**Pull request closed without merging**